integration-tests/emulated/src/mock/para_chains.rs

```diff
@@ -116,17 +110,17 @@ pub mod peregrine {
 }
 
 decl_test_parachains! {
-	pub struct Spiritnet {
+	pub struct SpiritnetParachain {
```

Comment:
Why?

Reply:
The macros and the generated structs changed with this new release. It is possible now, to directly import emulated chains. Those chains have to be initialized in the desired network. So the Spiritnet struct generated by `decl_test_parachain` is not usable in the tests anymore. I didn't want to rename all occurrences in the test files. Therefore, I renamed it here and initialized the needed struct in network module.